app/modules/profile/scripts/profile.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python
+"""
+extracts information about the profiles from system profiler
+"""
+
+import sys
+import os
+import subprocess
+import plistlib
+import json
```

Comment:
I added a check to see what version of OS X is being run and quit if it was less than 10.7. I believe 10.7 was the first to even support profiles.